pkg/logs/db_info.go

```diff
@@ -0,0 +1,7 @@
+package logs
+
+// DBInfo is logged when there is some info to log with the backend dependency
+type DBInfo struct {
+	Message string `logevent:"message,default=database-info"`
```

Comment:
We try to make our log messages as specific as possible so that they are identifiable more easily in the log output. My recommendation to most folks is to use these rules for operational data:

- Logging at less than an error level indicates a lack of confidence that the code functions correctly. Convert DEBUG and INFO logs to tests.
- For every unrecoverable exception there is a metric that contributes to an alert/alarm.
- For every unrecoverable exception there is one, and only one, error log that provides the context into that state.
- For every error log emitted by a service there is one, and only one, location within the code from which that log text originates.
- All related logs have a single identifier by which they may be isolated when viewing.

None of these are true in all situations and am happy to chat about them in more depth. I share them more to illustrate that we've preferred more specific logging wherever possible. For example, instead of having a `DBError` we might have `DBConnectionFailed` or instead of `DBInfo` we might have `DBConnectionSucceeded` where each has their own, specific message. We've found it helps a lot during incidents.